src/utils.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line no-control-regex
```

Comment:
Why is this required? Also this file doesn't seem to be properly formatted.

Reply:
https://stackoverflow.com/questions/49743842/javascript-unexpected-control-characters-in-regular-expression

It's needed of the ansi escaping

Reply:
I was referring to the eslint rule disable? Does this regex cause any issue?

Reply:
No issues, it does the job it's supposed to do 

> Control characters are special, invisible characters in the ASCII range 0-31. These characters are rarely used in JavaScript strings so a regular expression containing elements that explicitly match these characters is most likely a mistake.

From the eslint docs, but in this instance it's not a mistake